INSTALL.md

```diff
@@ -63,207 +63,286 @@ cfgov assets
 cfgov start django
 ```
 
+
 ## Stand-alone installation
```

Comment:
This title doesn't match the anchor link at the top. Can you change that to #stand-alone-installation?